### DISCUSSION TITLE
Has there been any update in column order/type etc since June 2022 submission

### DISCUSSION THREAD

**Discussion by mano-at-sdsc** [nih-cfde/published-documentation#387] [Q&A]
I usually go to every table description here (https://github.com/nih-cfde/published-documentation/wiki/C2M2-Table-Summary) to find if there as any change in table schema, but if it is known that there was no change since June 2022 submission, then I can skip this step in my internal checking. Thanks.

**Comment by jessicalumian** [comment]
Sorry for the delay in response. There has been no change to the data model since the June 2022 submission.

**Comment by mano-at-sdsc** [comment]
No worries. I successfully submitted and approved the metadata for MW. Thanks.

**Reply by mano-at-sdsc** [reply]
Dear Jessica:
There has been some change in the schema for some of the tables, as stated in past few OWG meetings and documented in the agenda document there. Is it possible to put a note or just * the tables with change in schema on this page: https://github.com/nih-cfde/published-documentation/wiki/C2M2-Table-Summary . This helps us pay more attention to these tables as we prepare them. Thanks.

**Comment by jonathancrabtree** [comment]
Hi @mano-at-sdsc, here are the C2M2 release notes, which I think might be what you're looking for (specifically the "December 2022 Release"):

[https://docs.nih-cfde.org/en/latest/c2m2/draft-C2M2_specification/#c2m2-release-details](https://docs.nih-cfde.org/en/latest/c2m2/draft-C2M2_specification/#c2m2-release-details)

**Reply by mano-at-sdsc** [reply]
Thanks. This is exactly what I was looking for. I have now noted the URLs.
